torch/csrc/jit/runtime/serialized_shape_function_registry.cpp

```diff
@@ -2678,17 +2875,9 @@ def native_batch_norm(input: List[int],
   return out
 
 def nonzero_lower_bound(input: List[int]) -> List[int]:
-  if torch.ge(torch.len(input), 1):
-    pass
```

Comment:
Hmm, do you need to rebase ? im surprised this is here

Reply:
I have rebased it, but I am still getting this.